stl/inc/vector

```diff
@@ -1755,54 +1755,6 @@ template <class _Iter, class _Alloc = allocator<_Iter_value_t<_Iter>>,
 vector(_Iter, _Iter, _Alloc = _Alloc()) -> vector<_Iter_value_t<_Iter>, _Alloc>;
 #endif // _HAS_CXX17
 
-template <class _Ty, class _Alloc>

```

Comment:
Would it be better to add a forward declaration of `vector<bool>` here so these declarations could stay in synopsis order?

Reply:
Sure.